packages/mikado_reborn/src/components/Button/index.ts

```diff
@@ -1,5 +1,5 @@
-import MkrContainedButton from './Contained/ContainedButton';
-import MkrOutlinedButton from './Outlined/OutlinedButton';
-import MkrTextButton from './Text/TextButton';
+import MkrContainedButton from './Contained/ContainedButton.vue';
+import MkrOutlinedButton from './Outlined/OutlinedButton.vue';
+import MkrTextButton from './Text/TextButton.vue';
 
 export { MkrContainedButton, MkrOutlinedButton, MkrTextButton };
```

Comment:
Je crois que tu peux faire ça directement

```suggestion
export MkrContainedButton from './Contained/ContainedButton.vue';
export MkrOutlinedButton from './Outlined/OutlinedButton.vue';
export MkrTextButton from './Text/TextButton.vue';
```